### ISSUE TITLE
render() doesn't correctly support relative paths when not using the current directory

### ISSUE THREAD

**Issue by DavidNeiman** [xflr6/graphviz#71]
To Reproduce:
create a graph with an html label that shows an image with a relative path (e.g. resources/1.jpg)

try to render the file not in the current directory, but rather in a subdirectory with resources being under that subdirectory
e.g. "graph/test.svg" with "graph/resources/1.jpg"
the code will look for "resources" relatively to current directory rather than relative to "graph" and will fail

note: this happens regardless of whether the "directory" argument  is used in files.render() or not

Suspected reason:
backend.py ->render() does not pass working directory of the destination file when calling run()
this makes both bakend.render and file.render unusable in the problematic use-case

Suggested solution:
* the easiest way is to:
  1) pass cwd=os.path.dirname(filepath) when calling run() in backend.render() - this will automatically be passed to subprocess.Popen via **kwargs
  2)  only using os.path.basename(filepath) when calling command()

**Comment by xflr6** [comment]
Thanks, to reproduce (same if the source file and the image  are in the same subdir):
```python
In [1]: import os
   ...: import pathlib
   ...: import tempfile
   ...: import urllib.request
   ...: 
   ...: import  graphviz
   ...: 
   ...: DIR = pathlib.Path('subdir')
   ...: PNG = ('https://en.wikipedia.org/static/images/project-logos/'
   ...:        'enwiki.png')
   ...: 
   ...: os.chdir(tempfile.mkdtemp())
   ...: DIR.mkdir()
   ...: png, _ = urllib.request.urlretrieve(PNG,
   ...:                                     DIR / PNG.rpartition('/')[2])
   ...: png = pathlib.Path(png)
   ...: 
   ...: g = graphviz.Graph(directory=DIR)
   ...: #g.node('spam', label='', image=str(png))
   ...: g.node('spam', label='', image=png.name)
   ...: g.view()
Warning: No such file or directory while opening enwiki.png
Warning: No or improper image="enwiki.png" for node "spam"
Out[1]: 'subdir\\Graph.gv.pdf'
```
Rendering from the directory of the source file will allow to use relative image paths properly.
This will break currently working (but odd?) use cases of specifying image paths relative to the caller directory (cf. the commented out line  above).

**Comment by xflr6** [comment]
Can you try if this branch works for you? https://github.com/xflr6/graphviz/tree/relative_paths

**Comment by xflr6** [comment]
Added to `master` via 8e92a7f43b093b546b68f3231e40402df40b5cc0 and  8d9a39793aa5ec9dc33ca4ae5c4424f548e814e7, closing.

**Comment by DavidNeiman** [comment]
Works like a charm!
Thanks!
I'd love to know if a new release is planned in the near future.

**Comment by xflr6** [comment]
Thanks, released [graphviz 0.11](https://pypi.org/project/graphviz/0.11/).
